Assets/Scripts/LevelTransitions/LevelTranstionController.cs

```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+using OperationBlackwell.Core;
+
+namespace OperationBlackwell.LevelTransitions {
+	public class LevelTranstionController : MonoBehaviour {
+		public static LevelTranstionController instance { get; private set; }
+		
+		[SerializeField] private BaseCutsceneController cutsceneController_;
+		[SerializeField] private Plane plane_;
+
+		bool test = false;
+
+		private void Awake() {
+			if(instance == null) {
+				instance = this;
+			} else {
+				Destroy(gameObject);
+			}
+		}
+
+		private void Update() {
+			if(!test) {
```

Comment:
I'm not sure if the name test is best suitable. It seems like this was actually just a test and you might have forgotten to change it.

Reply:
Well yes it is a test, but I did not forget about it. It is a test to display a placeholder cutscene during the transition, as the design team has only stated it might want a cutscene here it is just a test to see if that works.